aeon/dj_pipeline/README.md

```diff
@@ -1,93 +1,89 @@
+
++ `lookup`-tier tables (gray): Define static reference information
++ `manual`-tier tables (green): Contain user-inputted data
++ `imported`-tier tables (purple): Store data ingested from external sources
++ `computed`-tier tables (red): Represent results of automated computations
+
+Data flows through the pipeline in a top-down manner, driven by a combination of ingestion and computation routines. This layered organization facilitates efficient data processing and modular analysis.
 
-Such is also the way the data flows through the pipeline, by a combination of ingestion and
-computation routines.
 
 ## Core tables
 
 #### Experiment and data acquisition
 
-1. `Experiment` - the `aquisition.Experiment` table stores meta information about the experiments
++ `Experiment` - the `aquisition.Experiment` table stores meta information about the experiments
 done in Project Aeon, with secondary information such as the lab/room the experiment is carried out,
 which animals participating, the directory storing the raw data, etc.
 
-2. `Epoch` - A recording period reflecting on/off of the hardware acquisition system.
++ `Epoch` - A recording period reflecting on/off of the hardware acquisition system.
 The `aquisition.Epoch` table records all acquisition epochs and their associated configuration for
 any particular experiment (in the above `aquisition.Experiment` table).
 
-3.`Chunk` - the raw data are acquired by Bonsai and stored as
++ `Chunk` - the raw data are acquired by Bonsai and stored as
 a collection of files every one hour - we call this one-hour a time chunk.
 The `aquisition.Chunk` table records all time chunks and their associated raw data files for
 any particular experiment (in the above `aquisition.Experiment` table). A chunk must belong to one epoch.
 
-#### Devices
-
-5. `ExperimentCamera` - the cameras and associated specifications used for this experiment -
-e.g. camera serial number, frame rate, location, time of installation and removal, etc.
-
-6. `ExperimentFoodPatch` - the food-patches and associated specifications used for this experiment -
-e.g. patch serial number, sampling rate of the wheel, location, time of installation and removal, etc.
-
-7. `ExperimentWeightScale` - the scales for measuring animal weights, usually placed at the nest, one per nest
-
-#### Data streams
-
-8. `FoodPatchEvent` - all events (e.g. pellet triggered, pellet delivered, etc.)
-from a particular `ExperimentFoodPatch`
+#### Position data
 
-9. `FoodPatchWheel` - wheel data (angle, intensity) from a particular `ExperimentFoodPatch`
++ `qc.CameraQC` - quality control procedure applied to each `ExperimentCamera` (e.g. missing frame, etc.)
 
-10. `WheelState` - wheel states (threshold, d1, delta) associated with a given `ExperimentFoodPatch`
++ `tracking.SLEAPTracking` - position tracking for object(s), from a particular `VideoSource` per chunk
```

Comment:
This is good.
Please expand more on the 2 part tables
- `PoseIdentity` - identified Subject (i.e. Identity) and stores the name of the body part used as "anchor_part"
- `Part` - inferred x,y position over time for all body parts from SLEAP model